datadog_checks_dev/datadog_checks/dev/utils.py

```diff
@@ -18,6 +18,13 @@
 from .compat import FileNotFoundError, PermissionError
 from .structures import EnvVars
 
+if PY3:
+    from pathlib import Path
+    HOME = str(Path.home())
+else:
+    from os.path import expanduser
+    HOME = expanduser("~")
```

Comment:
We don't need this constant that will increase startup time, just use `expanduser` where it's needed